Lesson_5/Tusk_1.py

```diff
@@ -0,0 +1,15 @@
+text = input('Please enter some text:')
+
+for symbol in text:
+    if symbol.isdigit():
+        if (int(symbol) % 2) == 0:
```

Comment:
There is no need to use parentheses here, you can write conditions like this: `if int(symbol) % 2 == 0:`